mlir_graphblas/src/lib/GraphBLAS/GraphBLASUtils.cpp

```diff
@@ -0,0 +1,112 @@
+#include "mlir/IR/Attributes.h"
+#include "mlir/IR/BuiltinOps.h"
+#include "mlir/IR/BuiltinTypes.h"
+#include "mlir/Dialect/StandardOps/IR/Ops.h"
+#include "mlir/Dialect/SparseTensor/IR/SparseTensor.h"
+#include "mlir/Dialect/Tensor/IR/Tensor.h"
+
+using namespace std;
+using namespace mlir;
+using namespace mlir::sparse_tensor;
+
+// make CSR tensor type
+mlir::RankedTensorType getCSRTensorType(MLIRContext *context, Type valueType)
```

Comment:
I think we can replace `mlir::RankedTensorType` -> `RankedTensorType` because of `using namespace mlir;` above.